tracks/javascript/exercises/pascals-triangle/mentoring.md

```diff
@@ -1,47 +1,44 @@
 ### Reasonable solutions
 
 ```javascript
-function buildRow(result, element, index, array) {
-  return result.concat([element + (array[index + 1] || 0)])
+const buildRow = (result, element, index, array) => {
+  return [...result, element + (array[index + 1] || 0)];
 }
 
-function makeTriangle(levels) {
-  const result = [[1]]
-  for (let x = 1; x < levels; x += 1) {
-    result.push(result[x - 1].reduce(buildRow, [1]))
-  }
-  return result
-}
+export const rows = levels => {
+  if (levels === 0)
+    return [];
 
-export class Triangle {
-  constructor(levels) {
-    this.rows = makeTriangle(levels)
-  }
-
-  get lastRow() {
-    return this.rows[this.rows.length - 1]
+  const result = [[1]]
+  for (let x = 1; x < levels; x++) {
+    result.push(result[x - 1].reduce(buildRow, [1]));
   }
+  return result;
 }
 ```
 
 ```javascript
-export class Triangle {
-    constructor(size) {
-        this.rows = [[1]]
-
-        for (let y = 1; y < size; y++) {
-            this.lastRow = []
-
-            for (let x = 0; x <= y; x++) {
-                const up = this.rows[y - 1]
-                const left = up[x - 1] || 0
-                const right = up[x] || 0
-
-                this.lastRow.push(left + right)
-            }
-            this.rows.push(this.lastRow)
-        }
+export const rows = size => {
+  if (size === 0)
+    return [];
+
```

Comment:
```suggestion
  if (size === 0) {
    return [];
  }

```

Reply:
Yes, OK: I am using Guidelines which enforces braces simplification as much as we can, but it looks like it is not so frequent... I will adjust that!